hypervisor/include/public/acrn_common.h

```diff
@@ -107,6 +191,8 @@ union vhm_io_request {
  * Based on the rules above, a typical VHM request lifecycle should looks like
  * the following.
  *
+ * @verbatim
```

Comment:
How about this instead:
```
 * Based on the rules above, a typical VHM request lifecycle should look like
 * the following, assuming the initial state is FREE:
 *
 * @verbatim embed:rst:leading-asterisk
 *
 * +-----------------------+-------------------------+----------------------+
 * | SOS vCPU 0            | SOS vCPU x              | UOS vCPU y           |
 * +=======================+=========================+======================+
 * |                       |                         | **Hypervisor**:      |
 * |                       |                         |                      |
 * |                       |                         | - Fill in type,      |
 * |                       |                         |   addr, etc.         |
 * |                       |                         | - Pause UOS vCPU y   |
 * |                       |                         | - Set state to       |
 * |                       |                         |   PENDING **(a)**    |
 * |                       |                         | - Fire upcall to     |
 * |                       |                         |   SOS vCPU 0         |
 * |                       |                         |                      |
 * +-----------------------+-------------------------+----------------------+
 * | **VHM**:              |                         |                      |
 * |                       |                         |                      |
 * | - Scan for pending    |                         |                      |
 * |   requests            |                         |                      |
 * | - Set state to        |                         |                      |
 * |   PROCESSING **(b)**  |                         |                      |
 * | - Assign requests to  |                         |                      |
 * |   clients **(c)**     |                         |                      |
 * |                       |                         |                      |
 * +-----------------------+-------------------------+----------------------+
 * |                       | **Client**:             |                      |
 * |                       |                         |                      |
 * |                       | - Scan for assigned     |                      |
 * |                       |   requests              |                      |
 * |                       | - Handle the            |                      |
 * |                       |   requests **(d)**      |                      |
 * |                       | - Set state to COMPLETE |                      |
 * |                       | - Notify the hypervisor |                      |
 * |                       |                         |                      |
 * +-----------------------+-------------------------+----------------------+
 * |                       | **Hypervisor**:         |                      |
 * |                       |                         |                      |
 * |                       | - resume UOS vCPU y     |                      |
 * |                       |   **(e)**               |                      |
 * |                       |                         |                      |
 * +-----------------------+-------------------------+----------------------+
 * |                       |                         | **Hypervisor**:      |
 * |                       |                         |                      |
 * |                       |                         | - Post-work **(f)**  |
 * |                       |                         | - set state to FREE  |
 * |                       |                         |                      |
 * +-----------------------+-------------------------+----------------------+
 *
 * @endverbatim
```

Reply:
(and need a rebase to get the acrn.doxyfile merge conflicts fixed)

Reply:
Thanks for the suggestions. I've updated the doc and resolve the conflicts as well.